<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,6,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="GBOX_connection">
         <property name="title">
          <string>Connection</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
            <item>
             <widget class="QLabel" name="LBL_config">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Config</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="TXT_config">
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="reload_config_button_2">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>32</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial Black</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="LBL_IP">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>IP</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="TXT_IP">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="LBL_PORT">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>80</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>PORT</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="TXT_PORT">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="BTN_connect">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>45</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial Black</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Connect</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GBOX_info">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>103</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_19">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Freq</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>  Freq min  0 MHz  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>  Freq max  450 MHz  </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Power</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>  Power min  0  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>  Power max  1000  </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Phase</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>  Phase min  0  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string>  Phase max  360  </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HFrameLayout"/>
   </item>
   <item>
    <widget class="QLabel" name="LBL_status">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>reload_config_button_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>openConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>899</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TXT_config</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>readConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>942</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_connect</sender>
   <signal>clicked(bool)</signal>
   <receiver>Form</receiver>
   <slot>connectToServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openConfig()</slot>
  <slot>readConfig()</slot>
  <slot>connectToServer()</slot>
 </slots>
</ui>
